wpeverest/everest-forms
# Feature - Bricks Builder Comaptibility

### All Submissions:






### Changes proposed in this Pull Request:



Previously, There are no Bricks compatibility. This PR adds the compatibility.

### How to test the changes in this Pull Request:

1. Enable Bricks module.
2. Install Bricks theme from Appearence > Add New Theme > Upload Theme > Active ( Theme zip is in zira ticket [Link](https://themegrill.atlassian.net/browse/EVF-1244))
3. Add new page and edit with Bricks.
4. Search everest forms in widget section and add that widget.
5. Select form and you can customize form design.
6. Check whether form is working or not.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Bricks Builder Comaptibility
